### DISCUSSION TITLE
recursive reload config

### DISCUSSION THREAD

**Discussion by ExpandingMan** [qtile/qtile#3442] [Ideas] [answered]
It would be really nice to have functionality for reloading the config that checks the entire `.config/qtile` directory instead of just `config.py`.  At the moment, there is a huge disadvantage to splitting the config into multiple files because the config can't be reloaded in place.

Is there a hack for forcibly reloading the entire config that I can use right now?

**Comment by elParaguayo** [comment, marked as answer]
Like this: https://github.com/qtile/qtile/pull/3396 ?

**Comment by ExpandingMan** [comment]
Hm, it doesn't seem to work for me... is there some specific format I need?  I was doing this with files in `.config/qtile`, perhaps I need an additional nested directory?

**Reply by elParaguayo** [reply]
Are you using the git version of qtile? I think it should work on files in the same folder but word need to check.

**Reply by ExpandingMan** [reply]
Oh, I didn't realize how recent a commit that was!  Cool, I think I'll just wait for this to be tagged and switch my config then, thanks!

**Reply by elParaguayo** [reply]
No problem. We're hoping to do next release soon.
